src/content/docs/guides/data-shaping/convert-data-formats.mdx

```diff
@@ -0,0 +1,436 @@
+---
+title: Convert data formats
+---
+
+Data comes in many formats. Converting between formats is essential for
+integration, export, and interoperability. This guide shows you how to transform
+data between JSON, CSV, YAML, and other common formats using TQL's print
+functions.
+
+## Print to JSON
```

Comment:
The entire guide focuses on `print_*` functions, but those keep the data wrapped inside tenzir. I propose to switch to `write_*` operators, as I believe they are far more useful in practice.